lambda_functions/stac/stac.py

```diff
@@ -126,13 +138,13 @@ def stac_dataset(metadata_doc, item_abs_path, parent_abs_path):
     return stac_item
 
 
-def valid_coord_to_geojson(valid_coord):
+def valid_coord_to_geojson(valid_coord, spatial_reference):
```

Comment:
This should be renamed, it's no longer purely for the valid coordinates reprojection.

Also, this implementation will be buggy when used for reprojecting the `geo_ref_points`, since I think they are just a bounding box. See https://github.com/opendatacube/datacube-core/issues/537 for more information on this common error when reprojecting a vector geometry.